ansible/group_vars/nimbus-pyrmont-testing.yml

```diff
@@ -0,0 +1,9 @@
+---
+beacon_node_network: 'pyrmont'
+beacon_node_cont_tag: 'testing-{{ (hostname is search("large")) | ternary("large", "small") }}'
+dist_validators_layout:
+  "testing-large-01.aws-eu-central-1a.nimbus.pyrmont": { start: 4500, end: 7000 }
+  "testing-small-01.aws-eu-central-1a.nimbus.pyrmont": { start: 7000, end: 8000 }
+  "testing-small-02.aws-eu-central-1a.nimbus.pyrmont": { start: 8000, end: 9000 }
+  "testing-small-03.aws-eu-central-1a.nimbus.pyrmont": { start: 0, end: 0 }
+  "testing-small-04.aws-eu-central-1a.nimbus.pyrmont": { start: 0, end: 0 }
```

Comment:
shall we move one validator-less node to `unstable` instead?

Reply:
...and run one testing node with a single validator?

Reply:
We do have another node running a single validator.

My reasoning here was that these are the bootstrap nodes which have an important function for our users. We want slightly higher stability for them. I picked `testing` because this will give us more value as a canary release before deploying bootstrap node changes to mainnet.

Reply:
ah ok - yeah, we have a single-validator node, but no _testing_ single-validator node (for a/b of single validator behaviour) - we also have no _unstable_ 0-validator node.

Reply:
anyway, these are minor points, whatever works